polyfill.js

```diff
@@ -3,7 +3,7 @@
 // This polyfill requires: globalThis, BigInt & private fields
 ;(() => {
     // Math.abs does not support BigInt.
```

Comment:
Oh this. Math.abs doesn't support BigInt. Does there any solution or I have to write an abs function myself?

Reply:
There's no built-in function for this, but I think @ljharb was interested in a proposal to relax this restriction.

Reply:
I think all Math methods where it makes sense should accept all number types, yes.

Reply:
Oh, `Math.sign` should support bigint too. Is there a plan for it? @ljharb